packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpMvsApi.ts

```diff
@@ -128,12 +128,26 @@ export class FtpMvsApi extends AbstractFtpApi implements MainframeInteraction.IM
         const result = this.getDefaultResponse();
         const profile = this.checkedProfile();
 
+        const dsorg = dsAtrribute.apiResponse.items[0]?.dsorg;
+        const isPds = dsorg === "PO" || dsorg === "PO-E";
```

Comment:
I'm curious if there are a few other kind of PDS `dsorg`s that we may not be considering. Also, not sure if FTP does support them (like PO-L I think)

Reply:
That's a good question - we use this check in most places in the codebase, but I think we should try to cover all of the possible PDS `dsorg` values moving forward 😅